flexget/plugins/modify/sort_by_weight.py

```diff
@@ -123,7 +123,7 @@ def on_task_filter(self, task, config):
         if len(entries) < 2:
             return
         config = self.prepare_config(config)
-        log.info('sorting ´undecided´,´accepted´ entries by weight!')
+        log.info('sorting undecided, accepted entries by weight')
```

Comment:
Seems like unnecessary logging. You would expect it to run if you've enabled it...